src/pages/login/login.ts

```diff
@@ -13,19 +12,17 @@ import {ContactServicesProvider} from "../../providers/contact-services/contact-
   templateUrl: 'login.html',
 })
 export class LoginPage {
-  phoneNumber: string = '0655545546'; // Supprimer la valeur dans la version finale
+  phoneNumber: string = '4000000004'; // Supprimer la valeur dans la version finale
```

Comment:
why are you changing the default number ?

Reply:
fixed